<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldInfoBox</class>
 <widget class="QGroupBox" name="FieldInfoBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о поле</string>
  </property>
  <property name="title">
   <string>Информация о поле</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_Name">
     <property name="text">
      <string>Unnamed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ширина</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_Width">
     <property name="text">
      <string>000</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Высота</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_Height">
     <property name="text">
      <string>000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Кол-во ячеек всего</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_CellsTotal">
     <property name="text">
      <string>000000</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
